pytorch_lightning/profiler/pytorch/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pytorch_lightning.profiler.pytorch.pytorch_utils import RegisterRecordFunction
+from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_10
+
+if _TORCH_GREATER_EQUAL_1_10:
```

Comment:
This means only a single version would be present within the documentation.

Reply:
A bit confused here. 

I thought the new API was introduced in 1.8.1

```py
_TORCH_GREATER_EQUAL_1_8_1 and torch.profiler.kineto_available()
```

Reply:
the new API is still not in 1.10. Hopefully, it could be in 1.11.0 in next release.